dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05370ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,354 @@
+                    .loadResults();
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with absolute path container
+     * Should: Should does not anything
+     */
+    @Test
+    public void whenTemplateLayoutHasAbsolutePathShouldKeepThePathWithoutChange() throws IOException, DotDataException, DotSecurityException {
+
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(jsonDrawBody, "//" + host.getHostname());
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, "//" + host.getHostname()))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05370ChangeContainerPathToAbsolute task05370ChangeContainerPathToAbsolute =
+                new Task05370ChangeContainerPathToAbsolute();
+
+        task05370ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with relative path container
+     * Should: Should change the relative path by absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasRelativePathShouldTurnIntoAbsolutePath() throws DotDataException {
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(legacyHTMLLayout, "//" + host.getHostname());
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, ""))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05370ChangeContainerPathToAbsolute task05370ChangeContainerPathToAbsolute =
+                new Task05370ChangeContainerPathToAbsolute();
+
+        task05370ChangeContainerPathToAbsolute.executeUpgrade();
+        CacheLocator.getTemplateCache().remove(template.getInode());
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with absolute path container
+     * Should: Should keep the absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasAbsolutePath() throws DotDataException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5990100)